libpod/volume.go

```diff
@@ -139,6 +139,15 @@ func (v *Volume) MountPoint() (string, error) {
 	return v.mountPoint(), nil
 }
 
+// MountCount returns the volume's mountcount on the host from state
+// Useful in determining if volume is using plugin or a filesystem mount and its mount
+func (v *Volume) MountCount() (uint, error) {
+	if err := v.update(); err != nil {
```

Comment:
Need to lock the volume (`v.lock.Lock()`) before this

Reply:
@mheon added in latest commit.